src/components/copy-tooltip/CopyTooltip.ios.js

```diff
@@ -20,12 +20,6 @@ class CopyTooltip extends PureComponent {
     tooltipText: 'Copy',
   };
 
-  componentDidUpdate = () => {
```

Comment:
I haven't observed any difference. Also, on this screen (Profile) we're using native sheets and  they don't have these callback anyway so it's no-op.